.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish to NPM
+on:
+  release:
+    types: [published]
+    branches:
```

Comment:
Wait, this line should line should be deleted, haha

Reply:
L5?

Reply:
yes